### ISSUE TITLE
Blog post: How we approach documentation at MeiliSearch

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#129]

**Comment by bidoubiwa** [comment]
Do you mean the philosophy?
Those were my guidelines:
- Examples of fast setup to hook everyone in trying quickly and be convinced
- Extremely simple step by step explanation so that everyone can follow without stumbling on a paragraph that would be technically hard to understand
- Use of very simple words so that every basic knowledge of English is enough to understand our documentation
- Do not assume something is easy because it became easy for you. (terminal, curl, HTML)

Of course, you can change them however you want. I do not think my view is the only acceptable one haha.

**Comment by fharper** [comment]
Yes, our philosophy mostly, as what we expect in terms of quality level will be more in the contributing documentation. I'll let @react-learner add more details if needed.

**Comment by react-learner** [comment]
It's two birds w/ one stone: defining our documentation philosophy will help the team grow in the future, and might also be interesting to others in the industry :) 

I think this particular blog post will not only be about our philosophy, but also about challenges we've faced and what we aspire to in the future. Consider that truly open-source documentation may not exist yet—what would that look like? How can we lead the charge?

**Comment by fharper** [comment]
It would be nice to have, but not a priority right now. Maybe the new Technical Writer can write something about it, what do you think @react-learner ?

**Comment by react-learner** [comment]
Agreed, not a priority. I'm sure the move to Docusaurus will also provide some interesting experiences to write about in the future.

**Comment by ferdi05** [comment]
is this still relevant @dichotommy ?
